### DISCUSSION TITLE
Create path from outline SVG

### DISCUSSION THREAD

**Discussion by matzeeable** [fabricjs/fabric.js#7942] [Q&A] [answered]
Hey!

I have successfully imported an SVG to my canvas with the following code:

```
fabric.loadSVGFromURL("./my-svg.svg", function (objects, options) {
    const grouped = fabric.util.groupSVGElements(objects, options);
    grouped.scaleToWidth(200).set({
        left: 30,
        top: 30
    });
});
``` 

Now I want to calculate a path from the outline / silhouette of the complete SVG as I would like to create a border "around the complete" object. Is something like this possible with fabric?

**Comment by ShaMan123** [comment, marked as answer]
you can apply `stroke` using the `strokeWidth` `stroke` etc. properties.
the border props control what is displayed one you select the object.
